test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -848,3 +885,17 @@ test
     // verify login_hint has been appended to the app link url
     await t.expect(getPageUrl()).contains('login_hint='+encodeURIComponent(username));
   });
+
+test
+  .requestHooks(loopbackEarlyCancelChallengeErrorLogger, loopbackEarlyCancelChallengeErrorMock)('expect no error message in view', async t => {
+    const deviceChallengePollPageObject = await setup(t);
+    await t.expect(deviceChallengePollPageObject.getFormTitle()).eql('Verifying your identity');
+    
+    await deviceChallengePollPageObject.clickCancelAndGoBackLink();
+    await t.wait(EARLY_CANCEL_CHALLENGE_REQUEST_WAIT_TIME + 1000); // wait for delayed challenge request
```

Comment:
Add an assertion to ensure that the `POST /challenge/` request was actually made . You should be able to get that information from `loopbackEarlyCancelChallengeErrorLogger`.

Reply:
added at the end of test